metadata.json

```diff
@@ -1,14 +1,15 @@
 {
   "name": "simp-libkv",
-  "version": "0.5.0",
+  "version": "1.0.0",
```

Comment:
So, small difference in RE for this module; we don't bump the actual version number until we cut a release off of develop. Then we gen the CHANGELOG from git log.

It's something we started with this module back when we were having REC discussions, and it's continued because that's the model we had kind of wanted.

Reply:
Sounds good! I'll push it back.